db/migrate/20150720223731_create_memberships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMemberships < ActiveRecord::Migration
+  def change
+    create_table :memberships do |t|
+      t.references :user, index: true, foreign_key: true, null: false
+      t.references :game, index: true, foreign_key: true, null: false
+
+      t.timestamps null: false
+    end
+    add_index :memberships, [ :user_id, :game_id ], unique: true
```

Comment:
do you also want to index on just `:game_id` for fast lookups of the members of a particular game?